common/components/add-another/template.njk

```diff
@@ -10,6 +10,8 @@
 {% set items = params.items %}
 
 <div class="moj-add-another">
+  {{ params.hint.html | safe if params.hint.html else params.hint.text }}
```

Comment:
Should this not be a heading or even, a legend in another uber fieldset that wraps all the other ones?

Reply:
That's probably a good point. Although, in our current use it does get wrapped in a `p`.

If we wrap this component in a heading that's probably not ideal...